README.md

```diff
@@ -75,51 +75,83 @@ Reworked into these six RFCs:
-        or if we're really-really-careful-yes-we- really-mean-it-this-time)
+The Golden Rule **MUST NOT** be broken apart from in *exceptional* circumstances, such as for legal reasons.
 
+The versioning stategy has changed, existing endpoints using the 'legacy' versioning will continue to be supported until deprecated.
 
-*Content Negotiation* 
+#####New Versioning Strategy
+
+All new endpoints **MUST** have a version number in their URL directly beneath the resource name.
+
+    e.g. /documents/v1/<id>
+
+Endpoints that include a version within the URL **MUST NOT** include a version within the media type.
+
+    application/vnd.mendeley-<RESOURCE_TYPE>+json
+    e.g. application/vnd.mendeley-document+json
+    
+Clients **MUST** send a non-null and non-wildcard Accept header for endpoints that produce content. Clients **MUST** also send a non-null Content-Type header for endpoints that consume content.
+
+####Legacy Versioning Strategy
 
 Use HTTP's content negotiation mechanism for versioning. Custom media types are used in the API to let the consumers choose the format of the data they wish to receive. This is done by adding the `Accept` header when you make a request. Media types are specific to resources, allowing them to change independently.  
 
-	application/vnd.mendeley-<RESOURCE_TYPE>.<VERSION>+json
-	
-	e.g. application/vnd.mendeley-document.1+json.  The "1" here is the version number of the representation.
-	
-	
+    /<RESOURCE_TYPE>s/<id>
+    application/vnd.mendeley-<RESOURCE_TYPE>.<VERSION>+json
+    
+    e.g. /documents/<id>
+    application/vnd.mendeley-document.1+json.  The "1" here is the version number of the representation.
+
 Both client and servers are responsible for negotiation to be successful. 
 
 **Client Responsibilities** 	
 
 * Clients **MUST** send an appropriate resource Media type in the header 
 		* e.g. Accept: application/vnd.mendeley-document.1+json
 
-
-**Server Responsibilities**   	
+**Server Responsibilities**
 
 * Servers **MUST** respond with a Content-Type header 
-		* e.g. Content-Type: application/vnd.mendeley-document.1+json 	
-	
+		* e.g. Content-Type: application/vnd.mendeley-document.1+json
+
+Endpoints that include a version within the media type **MUST NOT** include a version within the URL.
+
+###Release Lifecycle
+
+All endpoints **MUST** follow the Release Lifecycle, each phase of the lifecycle has a set of rules associated with it.
+
+####Alpha
+During this phase of development endpoints may change without giving notice to internal/external clients. Properties can be added and removed, media types modified and the entire endpoint renamed or removed. 
+
+####Beta
+When an endpoint is promoted to the Beta phase clients using the endpoint **MUST** be notified of breaking changes and reasonable notice given to allow development teams to respond to the proposed changes; non-breaking changes can be made at any time without notice.
```

Comment:
I think this is too strong. We can't be certain of identifying all clients using the endpoint. Clients are notified on a best-effort basis.